core/src/main/resources/jenkins/security/ApiTokenProperty/config.jelly

```diff
@@ -100,7 +100,7 @@ THE SOFTWARE.
                                    data-message-if-legacy-revoked="${%RevokedToken}"
                                    data-confirm="${%ConfirmRevokeSingle}"
                                    data-target-url="${descriptor.descriptorFullUrl}/revoke">
-                                    <l:icon class="icon-text-error icon-sm"/>
+                                    <l:icon class="icon-text-error icon-sm" alt="${%RevokeToken}" />
```

Comment:
As you said, I abused the icon (because of the lack of pure delete icon in the core icon-set) but even with the `alt` I do not see any tooltip for this icon. (Tested with Chrome)

Are you ok to add also `title="${%RevokeToken}"` to have tooltip on that link ?

Reply:
Yes.